primitives/relayers/src/lib.rs

```diff
@@ -185,21 +185,21 @@ mod tests {
 	fn different_directions_are_using_different_accounts() {
 		assert_eq!(
 			PayRewardFromAccount::<(), H256>::rewards_account(RewardsAccountParams::new(
-				LaneId([0, 0, 0, 0]),
+				LaneId::new(1, 2),
 				*b"test",
 				RewardsAccountOwner::ThisChain
 			)),
-			hex_literal::hex!("62726170000000007465737400726577617264732d6163636f756e7400000000")
+			hex_literal::hex!("627261706087240931c736c98ccb5748ffe8b0a946fb1bb382db195ad826ec3e")
 				.into(),
 		);
 
 		assert_eq!(
 			PayRewardFromAccount::<(), H256>::rewards_account(RewardsAccountParams::new(
-				LaneId([0, 0, 0, 0]),
+				LaneId::new(1, 2),
 				*b"test",
 				RewardsAccountOwner::BridgedChain
 			)),
-			hex_literal::hex!("62726170000000007465737401726577617264732d6163636f756e7400000000")
+			hex_literal::hex!("627261706087240931c736c98ccb5748ffe8b0a946fb1bb382db195ad826ec3e")
```

Comment:
This change is obviously wrong - accounts need to be different. I'm gonna fix it + add note to description

Reply:
Fixed